resources/osl_awstats_site.rb

```diff
@@ -4,15 +4,27 @@
 
 default_action :create
 
-property :site_domain, String, name_property: true
-property :host_aliases, Array, default: []
-property :log_file, String, default: '*.log'
-property :append_suffix, [true, false], default: false
-property :log_suffix, String, default: '.log'
+property :log_file, String, default: '*'
+property :log_ext, String, default: '.log'
 property :append_date, [true, false], default: false
 property :date_format, String, default: '-%YYYY-2%MM-2%DD-2'
-property :only_files, Array, default: []
 property :vsftp_logs, [true, false], default: false
+property :site_domain, [String, Array], name_property: true
+property :host_aliases, [String, Array], default: ''
+property :log_format, String, default: %w(
+  %virtualname
+  %host
+  %other
+  %logname
+  %time1
+  %methodurl
+  %code
+  %bytesd
+  %refererquot
+  %uaquot
+  %other
+).join(' ')
```

Comment:
Move this to a helper method and then call it like this:
```suggestion
property :log_format, String, default: lazy { new_helper_method_name }
```